### PR TITLE
Add All my todos Files For Review

### PR DESCRIPTION
**Pull request by Alusp**
### IN THIS PR:
**I did the following,**

- Create a new empty repo.
- Copy my code from the "To Do list" project and commit it as one big chunk of code in a feature branch.
- Open a pull request.
- Ask you partner for a review.
- Share the pull request link with my partner.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.hello {
+  color: rgb(112, 189, 122);
```
Comment (Bateyjosue) [inline src/style.css]: Please, Consider creating a variable for the `color`  property as it appears more than twice 
 one line 18 and 71 for the BG, see line 28, and 54 as well to conform with the DRY rule.

**Review by Bateyjosue — CHANGES_REQUESTED** [review]
# Requested Changes ♻️ 

Mr Simpson

Hi @Alusp , 

Good job so far! 👍🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ JS Best Practice
✅ HTML and CSS Best Practive
✅ DRY, KISS, and YAGNI rules.


## Required Changes ♻️
Man your work is waouh, Good job so far
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alusp** [comment]
Alright, it makes a lot of sense, thanks very much.

**Review by Bateyjosue — APPROVED** [review]
## Hello @Alusp     :wave: ,
Call me Josh your reviewer for this PR :man_beard:  Thank you for making the changes that were requested by the previous reviewer. :clap:

Approved

### Great review!
- :heavy_check_mark: Your repo is professional
- :heavy_check_mark:  Good summary description 
- :heavy_check_mark:  No Linter errors 
- :heavy_check_mark:  use Of ES6 Module
- :heavy_check_mark:  Js Best practice
- :heavy_check_mark:  HTML & CSS best practice




Your project is complete! Congratulations! 🎉 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Bateyjosue** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nemwel-Boniface** [comment]
Ehem @Bateyjosue I can see I was also tagged :eyes: ?

**Pull request merged**